Cargo.toml

```diff
@@ -68,6 +68,9 @@ zstd = { version = "0.10.0", features = ["bindgen", "zstdmt"], default-features
 [target.'cfg(any(target_os = "macos", target_os = "windows"))'.dependencies]
 guess_host_triple = "0.1.3"
 
+[target.'cfg(windows)'.dependencies]
+mimalloc = { version = "0.1.29", default-features = false}
```

Comment:
I think it is better to make it an optional feature for now and enable it in our CI if it proves to be a problem later.